docs/guides/Getting-Started/getting-started/marketplace-overview.md

```diff
@@ -0,0 +1,185 @@
+Before sending suggestions, the seller can check if the SKU already exists in the marketplace. If it does not exist, the **Get SKU bindings by SKU ID** call should be made after sending the suggestion to notify the marketplace that the SKU in question is a new suggestion.
+
+- GET - [Get SKU Bindings by SKU ID](https://developers.vtex.com/docs/api-reference/sku-bindings-api#get-/catalog/pvt/skusellers/-skuId-)
+- PUT - [Send SKU Suggestion](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions#put-/suggestions/-sellerId-/-sellerSkuId-)
+
+### Marketplace: Deleting Suggestions
+
+If the marketplace wants to remove a submitted suggestion, they should use this call to delete the suggestion of a specific SKU.
+
+- DELETE - [Delete SKU Suggestion](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions#delete-/suggestions/-sellerId-/-sellerSkuId-)
+
+## <a name="Cataloging-offers"></a>3. Cataloging offers
+
+When the seller sends a product suggestion to the marketplace with configured price and inventory information, after approval by the marketplace, this suggestion becomes an offer. However, before approving a seller's suggestion, it is important to ensure that the product information sent corresponds to the business strategy of the marketplace. When cataloging offers, consider the quality of information sent by the seller, such as price, product description and image.
+
+Cataloging can be performed [manually](https://help.vtex.com/en/tutorial/sugerindo-e-aprovando-skus--tutorials_396#opcoes-de-catalogacao) or automatically by [VTEX matcher](https://help.vtex.com/en/tutorial/entendendo-a-pontuacao-do-vtex-matcher) and/or by external matchers. VTEX matcher is a default tool used by marketplaces to analyze offers submitted by sellers and accelerate their cataloging process.
+
+### Marketplace: Matching
+
+When matching received suggestions, the marketplace can perform this action either in bulk or individually.
+
+- PUT - [Match Received SKUs individually](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions#put-/suggestions/-sellerId-/-sellerskuid-/versions/-version-/matches/-matchid-)
+- PUT - [Match Multiple Received SKUs](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions#put-/suggestions/matches/action/-actionName-)
+
+### Marketplace: Auto Approve
+
+Received SKUs will be approved automatically, regardless of their [Matcher score](https://help.vtex.com/en/tutorial/entendendo-a-pontuacao-do-vtex-matcher--tutorials_424), if the auto approve feature is activated.
+
+- PUT - [Active autoApprove in the Marketplace's Account](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions/#put-/suggestions/configuration/autoapproval/toggle): Will be used by the marketplace to activate the auto Approve feature with the default parameters for all received SKU modules, so that all sent SKUs are automatically approved.
```

Comment:
```suggestion
- PUT - [Activate autoApprove in the Marketplace's Account](https://developers.vtex.com/docs/api-reference/marketplace-apis-suggestions/#put-/suggestions/configuration/autoapproval/toggle): Will be used by the marketplace to activate the auto Approve feature with the default parameters for all received SKU modules, so that all sent SKUs are automatically approved.
```